src/app/components/Figure/index.jsx

```diff
@@ -15,7 +16,7 @@ const renderCopyright = copyright => {
 };
 
 const renderCaption = caption =>
-  caption ? <Caption>{caption}</Caption> : null;
+  caption ? <Text text={caption} paragraphOverride={Caption} /> : null;
```

Comment:
Unless I'm mistaken we still need to render a Caption component as it includes the offscreen text but with the text (including markdown and style overrides) being passed as a child. EG:
```
<Caption>
  <Text text={caption} />
</ Caption>
```

However I'm currently unsure how this would change the styling of the markdown